python/cudf/cudf/core/series.py

```diff
@@ -1783,8 +1783,9 @@ def cumsum(self, axis=0, skipna=True):
         """Compute the cumulative sum of the series"""
         assert axis in (None, 0) and skipna is True
 
-        # pandas always returns int64 dtype if original dtype is int
-        if np.issubdtype(self.dtype, np.integer):
+        # pandas always returns int64 dtype if original dtype is int or `bool`
+        if np.issubdtype(self.dtype, np.integer) or \
+                np.issubdtype(self.dtype, np.bool):
```

Comment:
Should this use `np.bool_`? `np.float32` and `np.float64` are sub dtypes of `np.bool`, but not `np.bool_`. This may be why the tests are failing.